src/main/java/com/redhat/cloud/notifications/templates/Policies.java

```diff
@@ -0,0 +1,17 @@
+package com.redhat.cloud.notifications.templates;
+
+import io.quarkus.qute.TemplateInstance;
+import io.quarkus.qute.api.CheckedTemplate;
+
+
+public class Policies {
```

Comment:
I think we should make this (not necessarily now) more generic.
At the end each template has a header and a body

Reply:
Agree. In the future we can pull it out of there and put the real templates in the database. An intermediary step would be to have files like : `appName + templateType` where templateType is: instantEmailTitle, instantEmailBody, dailyEm....